### ISSUE TITLE
On Strapi Build: Error: Cannot find module '@vue/cli-plugin-babel/preset'

### ISSUE THREAD

**Issue by hunterInt** [strapi/strapi#10201]
Pretty much no matter the version (I've tried half a dozen different ones) when I run `strapi build` I get this error:
```
Building your admin UI with development configuration ...

✖ Webpack
  Compiled with some errors in 644.81ms

Error: ./.cache/admin/src/app.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@vue/cli-plugin-babel/preset' from '/workspace'
    at Function.resolveSync [as sync] (/workspace/node_modules/resolve/lib/sync.js:102:15)
    at resolveStandardizedName (/workspace/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePreset (/workspace/node_modules/@babel/core/lib/config/files/plugins.js:58:10)
    at loadPreset (/workspace/node_modules/@babel/core/lib/config/files/plugins.js:77:20)
    at createDescriptor (/workspace/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at /workspace/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
    at Array.map (<anonymous>)
    at createDescriptors (/workspace/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPresetDescriptors (/workspace/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
    at presets (/workspace/node_modules/@babel/core/lib/config/config-descriptors.js:47:19)
    at mergeChainOpts (/workspace/node_modules/@babel/core/lib/config/config-chain.js:320:26)
    at /workspace/node_modules/@babel/core/lib/config/config-chain.js:283:7
    at Generator.next (<anonymous>)
    at buildRootChain (/workspace/node_modules/@babel/core/lib/config/config-chain.js:90:27)
    at buildRootChain.next (<anonymous>)
    at loadPrivatePartialConfig (/workspace/node_modules/@babel/core/lib/config/partial.js:95:62)
    at /workspace/node_modules/strapi-admin/index.js:71:23
    at finalCallback (/workspace/node_modules/webpack/lib/Compiler.js:257:39)
    at /workspace/node_modules/webpack/lib/Compiler.js:273:13
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:24:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at onCompiled (/workspace/node_modules/webpack/lib/Compiler.js:271:21)
    at /workspace/node_modules/webpack/lib/Compiler.js:681:15
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at /workspace/node_modules/webpack/lib/Compiler.js:678:31
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at /workspace/node_modules/webpack/lib/Compilation.js:1423:35
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at /workspace/node_modules/webpack/lib/Compilation.js:1414:32
```

I recently setup a Strapi project using `npx create-strapi-app my-project --quickstart`. I then am able to run the `strapi start` command without any issues and I am able to use Strapi correctly (I've created collections and am able to use the API without issue). But when I tried cloning my project onto another PC and run `strapi build` I get that error above. Then I realized that even if I run `strapi build` inside the already working project I still get the same error (so `strapi start` is working on this machine, but not `strapi build`). 

Any ideas what I'm missing or what this bug is? Thanks!

- Node.js version: v14.16.0
- NPM version: 6.14.11
- Strapi version: 3.6.0
- Database: default 
- Operating system: Ubuntu and windows 10

**Comment by derrickmehaffy** [comment]
Can't reproduce on latest v3.6.1.

Also: 

> Error: Cannot find module '@vue/cli-plugin-babel/preset' from '/workspace'

We don't use Vue.js at all, so this is most likely not a Strapi issue but some kind of workspace issue, I've attached a yarn.lock file from a test project I just created and searching for that package in the lock, it doesn't exist. Without steps to reproduce this there isn't much we can do to help.


[yarn.lock.txt](https://github.com/strapi/strapi/files/6410806/yarn.lock.txt)
